### ISSUE TITLE
ListViewItem haven't name property value if there is image instead of text

### ISSUE THREAD

**Issue by VasylRudnitskyi** [dotnet/winforms#253]
.Net Framework 4.7.2

ListViewItem doesn't have value of accessible name property when it contains image, so screen reader cannot read it.

Repro
Create new windows form application .net 4.7.2
Create ImageList, add few images.
Create new Listview, add few list items with provided imageindex property.
Open Keros for Desktop and validate list.

Actual result: Name property is empty.
Expected result: Name property should to has a value.

![1](https://user-images.githubusercontent.com/45966665/50149481-fbecf700-02c3-11e9-9ce2-068dcd508f8c.png)
![2](https://user-images.githubusercontent.com/45966665/50149482-fbecf700-02c3-11e9-81bd-732605cc36c5.png)

**Comment by zsd4yr** [comment]
Looks like this is a framework bug, not a core bug -- @merriemcgaw what do you think?

**Comment by OliaG** [comment]
We should test it for Framework 4.8 and Core 3

**Comment by merriemcgaw** [comment]
@VasylRudnitskyi, this repo is for .NET Core 3 version of Windows Forms. From your report it appears that you got this for .NET 4.7.2. We've been making changes in the upcoming .NET 4.8. You should test with an [early access build of .NET 4.8](https://github.com/Microsoft/dotnet-framework-early-access) to see if the issue has been corrected. If it has not, please file an issue in that repo.

@M-Lipin , @bryukov - does this look like something that's just waiting to be ported to Core? Or do we have a fix here we need to make?

**Comment by bryukov** [comment]
@merriemcgaw I don't think we have a ready fix from 4.8 for this issue - at least I haven't found one.
